src/did.js

```diff
@@ -85,7 +94,7 @@ export class DID {
     }
   }
 
-  getAllOperations() {
+  async getAllOperations() {
```

Comment:
feels quirky to mark any function as `async` that doesn't explicitly `await` anything but i see the desire for consistency so 👍🏽

Reply:
yeah it's a bit odd, but personally I like to annotate all functions that return a `Promise` with `async` just so that it's super clear at a glance (and always guaranteed) that the thing being returned is in fact a `Promise`